docs/markdown/Wrap-dependency-system-manual.md

```diff
@@ -143,6 +143,34 @@ automatically by adding the following *(since 0.48.0)*:
 clone-recursive = true
 ```
 
+## Patch files
+
+*Since: 0.49.0*
+
+A list of local patch files can be provided in the `[patch-files]` section. They
```

Comment:
"patch-files" sounds extremely similar to patch_filename, which was, unwisely, not named "overlay_archive_filename". A while ago I lessened the confusion in the docs e.g. https://mesonbuild.com/Wrap-dependency-system-manual.html#wrapfile-with-meson-build-patch

> For historic reasons this is called a "patch", however, it serves as an overlay to add or replace files rather than modifying them. The file must be an archive;

But we didn't/couldn't change the wrap file format's key name.

In order to prevent even *more* confusion, we cannot use "patch-file". What about "diff-file"?

Reply:
IIRC (it was a long time ago) I opted for a new section specifically for avoiding confusion with `patch_filename`. Personally I still find it cleaner in a separate section.

Reply:
People will still just ask "do I add foo.patch to the *section* for patch files, or to the *key* for patch files?"